index.test.js

```diff
@@ -1,8 +1,10 @@
 const handler = require('./index').handler
+const now = new Date(1234567891234)
```

Comment:
Is there some significance to `Fri Feb 13 2009 18:31:31 GMT-0500 (Eastern Standard Time)` ? 😄